src/sentry/scim/endpoints/teams.py

```diff
@@ -387,7 +396,14 @@ def patch(self, request: Request, organization, team):
         }
         ```
         """
+
+        serializer = SCIMTeamPatchRequestSerializer(data=request.data)
```

Comment:
do you need to do this if we have `SCIMTeamPatchRequestSerializer` in the `@extend_schema` decorator before this method?

Reply:
@cathyteng17 Yea. This pattern was copied from this portion of the `PATCH` method for the users endpoint here: https://github.com/getsentry/sentry/blob/c152f33e94e99eae7761f78959cf166e0d7c5fec/src/sentry/scim/endpoints/members.py#L191-L219

 Without this, we don't propagate the validation errors.